src/ui/public/kibiter/menu/custom_menu.js

```diff
@@ -0,0 +1,56 @@
+
+export function customMenu($scope) {
```

Comment:
Rename this function to `drawKibiterMenu`, `displayKibiterMenu`, `renderKibiterMenu` or something similar.